suites/nautilus/ansible/tier-2_deploy_test-ceph-rpms.yaml

```diff
@@ -234,3 +234,22 @@ tests:
       polarion-id: CEPH-83573277
       module: purge_dashboard.py
       desc: Purges ceph dashboard
+
+  - test:
+      name: Ansible.log update
```

Comment:
based on the test name CI log file will be created so let's not provide any special char in test name remove . just keep it simple as ` anisble log collection`?

Reply:
> based on the test name CI log file will be created so let's not provide any special char in test name remove . just keep it simple as ` anisble log collection`?
@sunilangadi2 can we say `Ceph ansible log validation` instead?